content/en/guides/directory-structure/nuxt-config.md

```diff
@@ -327,6 +327,23 @@ See more on the [transition property](/guides/configuration-glossary/configura
 
 </base-alert>
 
+### Example
+
+<example-intro></example-intro>
+
+`nuxt.config.js` contains :
+
+- the `env` property to add the URL of our API.
```

Comment:
Could we use the `publicRuntimeConfig` instead of `env` for this example?

Reply:
you mean also change it in the docs: we only document env property and not publicRuntimeConfig / privateRuntimeConfig

https://nuxtjs.org/guides/directory-structure/nuxt-config#env

do we remove env from docs altogether or just add the extra ones to this page and use in example? @Atinux

Reply:
done